openbsd/OpenBSDProcess.c

```diff
@@ -209,7 +209,7 @@ void OpenBSDProcess_writeField(Process* this, RichString* str, ProcessField fiel
       Process_writeField(this, str, field);
       return;
    }
-   RichString_append(str, attr, buffer);
+   // RichString_append(str, attr, buffer); // <!-- unreachable
```

Comment:
for this one, it looks like the entire `OpenBSDProcess_writeField function` is there just as a placeholder to make it easier to add OpenBSD-specific fields in the future. The whole function could be removed and just replaced with `Process_writeField` in the definition of `OpenBSDProcess_class`. Did you add these comments to silence warnings in some specific platform?